### PR TITLE
pkg/updateengine: expose AttemptUpdate method

### PR DESCRIPTION
**Pull request by squeed**

### CODE REVIEW

**Comment by squeed** [comment]
@dghubble can I get a review?

File: pkg/updateengine/status.go [hunk 1]
```diff
@@ -18,6 +18,18 @@ import (
 	"fmt"
 )
 
+// The possible update statuses returned from the update engine
+const (
```
Comment (dghubble) [inline pkg/updateengine/status.go]: Include comment with link?

Comment (dghubble) [inline pkg/updateengine/status.go]: I see these are listed (only in comments) in pkg/constants/constants.go as well, though I kinda prefer them here. Pretty sure they correspond to current operation values so maybe specify that as well.

**Comment by lucab** [comment]
@dghubble as the requested change was comment-only and Casey will be away for some time, I just queued a commit on top of his branch adding an external reference. PTAL.

**Pull request merged**